provider/docker_test.go

```diff
@@ -6,11 +6,15 @@ import (
 	"testing"
 
 	"github.com/containous/traefik/types"
+	"github.com/davecgh/go-spew/spew"
+	dockerClient "github.com/docker/engine-api/client"
```

Comment:
Should be `dockerclient`. Go eschews camel case or even underscores in package names/aliases.